lib/client/api.go

```diff
@@ -353,6 +353,10 @@ type Config struct {
 
 	// Invited is a list of people invited to a session.
 	Invited []string
+
+	// ParticipantRequirements is set if debug information about participants requirements
+	// should be printed in moderated sessions.
+	ParticipantRequirements bool
```

Comment:
This name was a little confusing to me. How about `DisplayParticipantRequirements` instead?

Reply:
Sure